shacl12-sparql/index.html

```diff
@@ -1315,6 +1315,78 @@ <h3>Select Expressions</h3>
 					</div>
 				</aside>
 			</section>
+
+			<section id="EvalExpression">
+				<h3>Eval Expressions</h3>
+				<p>
+					A <a>node expression</a> that has a <a>value</a> for <code>sh:eval</code> is called an <dfn>eval expression</dfn> with the <a>function name</a>
+					<code>sh:EvalExpression</code>.
+				</p>
+				<p class="syntax">
+					<span data-syntax-rule="EvalExpression-syntax-eval">A node in an RDF graph is a <a>well-formed</a> <a>eval expression</a> if it is a <a>blank node</a>
+					that is the <a>subject</a> of exactly one <a>triple</a> with <code>sh:eval</code> as <a>predicate</a> and a <a>literal</a> as <a>object</a>
+					with <a>datatype</a> <code>xsd:string</code>.</span>
```

Comment:
I've suggested changes to make clearer what I think was the intended meaning (i.e., that the blank node could be the subject of multiple triples, but only one of these may have the specified predicate and object). It may be that I've misunderstood the intent, which will lead to different change suggestions (e.g., that the blank node may be the subject of only one triple, which has the specified predicate and object). My suggestion could be ignored, in favor of construction similar to that in lines 2312–2362.
```suggestion
					<span data-syntax-rule="EvalExpression-syntax-eval">A node in an RDF graph
					is a <a>well-formed</a> <a>eval expression</a>
					if it is a <a>blank node</a>
					that is the <a>subject</a> of exactly one <a>triple</a>
					with <code>sh:eval</code> as <a>predicate</a>
					and a <a>literal</a> with <a>datatype</a> <code>xsd:string</code> as <a>object</a>.</span>
```